components/collapsible-panel/collapsible-panel.js

```diff
@@ -321,6 +322,13 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 		this.type = 'default';
 		this.noSticky = false;
 		this._focused = false;
+		this._group = undefined;
+		this._groupSubscription = new EventSubscriberController(this, 'collapsible-panel-group', {
+			onSubscribe: (registry) => {
+				this._group = registry;
+				this._group._updatePanelAttributes();
```

Comment:
This handles the case where `updated` has already been called when the subscription kicks in.